agent/src/main/java/io/perfmark/agent/PerfMarkTransformer.java

```diff
@@ -257,7 +257,7 @@ static String deriveFileName(String className) {
     @Override
     public MethodVisitor visitMethod(
         int access, String name, String descriptor, String signature, String[] exceptions) {
-      return new PerfMarkMethodVisitor(
+      return new PerfMarkTransformer.PerfMarkMethodRewriter.PerfMarkMethodVisitor(
```

Comment:
This fixes the new errorprone warning.
https://gist.github.com/suztomo/a57c6e537c8f35795e365bbf71559ead#file-gistfile1-txt-L24

Reply:
Why not `return new PerfMarkMethodRewriter.PerfMarkMethodVisitor(`?

Reply:
Updated.